statistics/handle/handle.go

```diff
@@ -366,10 +378,8 @@ func (h *Handle) indexStatsFromStorage(row chunk.Row, table *statistics.Table, t
 	errorRate := statistics.ErrorRate{}
 	flag := row.GetInt64(8)
 	lastAnalyzePos := row.GetDatum(10, types.NewFieldType(mysql.TypeBlob))
-	if statistics.IsAnalyzed(flag) {
```

Comment:
This is another bug?

Reply:
Yes, the error rate should not be cleaned when using snapshot read.